### ISSUE TITLE
Improve the design of 辶 in TW/HK version

### ISSUE THREAD

**Issue by tamcy** [adobe-fonts/source-han-sans#50]
This issue refers to the actual rendering form of radical #162, i.e. the 辶 (U+8FB6) component in the Hong Kong/Taiwan version. This applies to all words that belong to the said radical, e.g.這遇過選遍 etc.

## Background

The design of 辶 in Source Han Sans is different from the the MoE standard and other font vendors. 

Here is an exerpt from MoE's "Chinese Character Fong Style" document:

![8fb6-1-moe](https://cloud.githubusercontent.com/assets/959433/3787851/50d4c5d8-1a3c-11e4-9342-9d46468b10f8.png)

And below illustrates how 辶 is drawn in Source Han Sans TWHK:

![8fb6-2-sourcehansans](https://cloud.githubusercontent.com/assets/959433/3787857/b5b4b742-1a3c-11e4-82b6-54e4511ff34b.png)

For comparison, here is how the same 辶 is rendered in other fonts:

![8fb6-3-3rdparty](https://cloud.githubusercontent.com/assets/959433/3787859/cf66bde8-1a3c-11e4-9b7c-326a0f01ad10.png)

One can immediately notice the stroke differences:

![8fb6-2](https://cloud.githubusercontent.com/assets/959433/3787880/15a850b8-1a3e-11e4-8035-f11d4b12c363.png)

It looks like the 辶 in Source Han Sans TWHK resembles that of MoE's Song style:

![8fb6-5-song](https://cloud.githubusercontent.com/assets/959433/3787864/ff2e8e98-1a3c-11e4-8b43-912814040305.png)

## The problem

To me, it is completely OK to be different and not 100% following the MoE standard. This issue isn't about not following the standard, but the design of this component which is not appealing:

a) The left side of the 辶 component looks a bit thin. It is more obvious when compared with other fonts. This gives me feel that the radical is separated from its surrounded component. The most obvious one is "導".

b) The second "乛" is shifted rightward compared to the first "乛". This causes a "stair" effect, which makes the component look unbalanced. (And it seems to be the reason why it feel it is "thin")

![8fb6-6-b](https://cloud.githubusercontent.com/assets/959433/3787866/62e3419a-1a3d-11e4-901a-f6d0c021d0c9.png)

c) In addition, since the upper "乛" is shifted to the left, the empty space is obvious when the upper part of the top-right component is smaller than the lower part (like 過).

![8fb6-6-c](https://cloud.githubusercontent.com/assets/959433/3787867/688894ba-1a3d-11e4-9a2f-580a3f8fe439.png)

d) The decoration on the corner of the second "乛" looks OK at small size. However, when it is viewed at a larger font size, it not only fails to enhance viewing experience, but gives an impression that the line was meant to be straight but accidentally cut/ground

![8fb6-6-d](https://cloud.githubusercontent.com/assets/959433/3787868/7d0f1a30-1a3d-11e4-9fd1-e0a44945543e.png)

e) Finally, the "spike" of the last stroke spoiled the smoothness of the left-slanting downward stroke. 

![8fb6-6-e](https://cloud.githubusercontent.com/assets/959433/3787869/8575a91e-1a3d-11e4-994c-cb265b583064.png)

## Closing

It seems that TWHK form of 辶 doesn’t play well with other components. It is a bit complicated and unbalanced, which makes me think the component is a hand-written.

It is hard to tell if Source Han Sans deliberately chose a different presentation in Hei(Gothic) style to differentiate itself from other vendors. While I agree the necessity to be different, my opinion is that the style in Source Han Sans lacks aesthetics.

It would be appreciated if the team could consider improving the look of this component.

**Comment by kenlunde** [comment]
Consolidated with Issue #99.

**Comment by kenlunde** [comment]
With regard to the 辶 component that is used for TW and HK glyphs, I am listing the affected characters here for the record:

236 TW (Big Five) characters: U+50C6 僆, U+55F9 嗹, U+5660 噠, U+5683 嚃, U+58DD 壝, U+5C0E 導, U+5DE1 巡, U+6165 慥, U+6425 搥, U+6453 摓, U+6459 摙, U+64BB 撻, U+64BE 撾, U+64FF 擿, U+65DE 旞, U+66B9 暹, U+69CC 槌, U+69E4 槤, U+6A96 檖, U+6A9B 檛, U+6AA4 檤, U+6ACF 櫏, U+6F23 漣, U+6FC4 濄, U+7021 瀡, U+7022 瀢, U+71A5 熥, U+71E7 燧, U+7489 璉, U+74A1 璡, U+74AD 璭, U+74B2 璲, U+74CB 瓋, U+7787 瞇, U+79AD 禭, U+7A5F 穟, U+7BF4 篴, U+7BF7 篷, U+7C09 簉, U+7C3B 簻, U+7C67 籧, U+7C69 籩, U+7CD9 糙, U+7E0B 縋, U+7E0C 縌, U+7E2B 縫, U+7E3A 縺, U+7E68 繨, U+7E78 繸, U+7E7E 繾, U+7FF4 翴, U+817F 腿, U+8187 膇, U+81BC 膼, U+84EA 蓪, U+84EB 蓫, U+84EC 蓬, U+84EE 蓮, U+8578 蕸, U+8596 薖, U+8598 薘, U+85B3 薳, U+85D7 藗, U+85E1 藡, U+8627 蘧, U+8781 螁, U+892A 褪, U+8933 褳, U+895A 襚, U+8B0E 謎, U+8B30 謰, U+8B74 譴, U+8E65 蹥, U+8E82 躂, U+8E9A 躚, U+8F4B 轋, U+8FBF 辿, U+8FC2 迂, U+8FC4 迄, U+8FC5 迅, U+8FC6 迆, U+8FC9 迉, U+8FCB 迋, U+8FCD 迍, U+8FCE 迎, U+8FD1 近, U+8FD2 迒, U+8FD3 迓, U+8FD4 返, U+8FD5 迕, U+8FD6 迖, U+8FD7 迗, U+8FE0 迠, U+8FE1 迡, U+8FE2 迢, U+8FE3 迣, U+8FE4 迤, U+8FE5 迥, U+8FE6 迦, U+8FE8 迨, U+8FEA 迪, U+8FEB 迫, U+8FED 迭, U+8FEE 迮, U+8FF0 述, U+8FF4 迴, U+8FF5 迵, U+8FF6 迶, U+8FF7 迷, U+8FF8 迸, U+8FFA 迺, U+8FFB 迻, U+8FFC 迼, U+8FFD 追, U+8FFE 迾, U+8FFF 迿, U+9000 退, U+9001 送, U+9002 适, U+9003 逃, U+9004 逄, U+9005 逅, U+9006 逆, U+900B 逋, U+900C 逌, U+900D 逍, U+900F 透, U+9010 逐, U+9011 逑, U+9014 途, U+9015 逕, U+9016 逖, U+9017 逗, U+9019 這, U+901A 通, U+901B 逛, U+901C 逜, U+901D 逝, U+901E 逞, U+901F 速, U+9020 造, U+9021 逡, U+9022 逢, U+9023 連, U+9024 逤, U+902D 逭, U+902E 逮, U+902F 逯, U+9031 週, U+9032 進, U+9034 逴, U+9035 逵, U+9036 逶, U+9038 逸, U+903C 逼, U+903D 逽, U+903E 逾, U+903F 逿, U+9041 遁, U+9042 遂, U+9044 遄, U+9047 遇, U+9049 遉, U+904A 遊, U+904B 運, U+904D 遍, U+904E 過, U+904F 遏, U+9050 遐, U+9051 遑, U+9052 遒, U+9053 道, U+9054 達, U+9055 違, U+9058 遘, U+9059 遙, U+905B 遛, U+905C 遜, U+905D 遝, U+905E 遞, U+9060 遠, U+9062 遢, U+9063 遣, U+9067 遧, U+9068 遨, U+9069 適, U+906B 遫, U+906D 遭, U+906E 遮, U+906F 遯, U+9070 遰, U+9072 遲, U+9073 遳, U+9074 遴, U+9075 遵, U+9076 遶, U+9077 遷, U+9078 選, U+9079 遹, U+907A 遺, U+907B 遻, U+907C 遼, U+907D 遽, U+907E 遾, U+907F 避, U+9080 邀, U+9081 邁, U+9082 邂, U+9083 邃, U+9084 還, U+9085 邅, U+9086 邆, U+9087 邇, U+9088 邈, U+908A 邊, U+908B 邋, U+908D 邍, U+908F 邏, U+9090 邐, U+919A 醚, U+939A 鎚, U+93C8 鏈, U+9429 鐩, U+9439 鐹, U+943D 鐽, U+945D 鑝, U+95E5 闥, U+96A7 隧, U+96A8 隨, U+9746 靆, U+97C3 韃, U+97C6 韆, U+9AD3 髓, U+9C31 鰱, U+9C41 鱁, and U+9D90 鶐.

84 HK (HKSCS-2016) characters: U+3493 㒓, U+35FB 㗻, U+3980 㦀, U+3F00 㼀, U+3FF9 㿹, U+489B 䢛, U+48AD 䢭, U+48AE 䢮, U+4A24 䨤, U+55F5 嗵, U+5873 塳, U+58B6 墶, U+6A0B 樋, U+6FBB 澻, U+6FBE 澾, U+717A 煺, U+71A2 熢, U+71F5 燵, U+78D3 磓, U+84BE 蒾, U+882D 蠭, U+8B62 譢, U+8B81 讁, U+8E46 蹆, U+8FB6 辶, U+8FB7 辷, U+8FB8 辸, U+8FB9 边, U+8FBA 辺, U+8FBB 辻, U+8FBC 込, U+8FBE 达, U+8FC1 迁, U+8FCA 迊, U+8FCC 迌, U+8FCF 迏, U+8FD0 运, U+8FDA 迚, U+8FF9 迹, U+9008 逈, U+9012 递, U+9033 逳, U+9037 逷, U+9046 遆, U+904C 遌, U+9056 遖, U+9061 遡, U+9064 遤, U+906C 遬, U+93E0 鏠, U+968F 随, U+9938 餸, U+9B14 鬔, U+2004E 𠁎, U+210B4 𡂴, U+210F4 𡃴, U+2143F 𡐿, U+2183B 𡠻, U+23262 𣉢, U+2408D 𤂍, U+24ABC 𤪼, U+26ED7 𦻗, U+26FBE 𦾾, U+273FF 𧏿, U+27785 𧞅, U+28468 𨑨, U+2846C 𨑬, U+28473 𨑳, U+28482 𨒂, U+28501 𨔁, U+2853C 𨔼, U+2853D 𨔽, U+2856C 𨕬, U+285E8 𨗨, U+285F4 𨗴, U+28600 𨘀, U+2860B 𨘋, U+28625 𨘥, U+2863B 𨘻, U+28BB9 𨮹, U+2915B 𩅛, U+29EAC 𩺬, U+29EF8 𩻸, and U+29F23 𩼣.

Two other characters: U+329C ㊜ and U+1F22B 🈫.

**Comment by hfhchan** [comment]
Hopefully the same changes will be seen in source-han-serif version 2.000 too ;)

**Comment by kenlunde** [comment]
@tamcy and others: I'd like to share with you what our designers came up with, in terms of the redesign of the 辶 component for the TW and HK glyphs:

![new-radical-162-for-tw-and-hk](https://cloud.githubusercontent.com/assets/3988788/26828557/eb73c5e0-4a76-11e7-9661-10cc6c400468.jpg)

**Comment by hfhchan** [comment]
👍🏻 stunning.

**Comment by tamcy** [comment]
Kudos to @kenlude and your team for your work. It definitely looks better than the original design. Just my 2 cents:

1. Is the angle of the first stroke finalized? The angle in KR/JP looks more natural to my eyes.
2. Seems the designer wants a more calligraphic feel? I wonder how the decorations in the 廴-like component happen in normal/regular weight.
3. When placed with 千, visually 千 looks too small and 辶 looks a bit too large or too heavy.

Of course that's just a layman's point of view. Thanks again for the update, can't wait to see the next release! 👍

**Comment by hfhchan** [comment]
I second that the a flatter first dot is more natural.

The dot could be as flat as this:
![image](https://user-images.githubusercontent.com/8191296/27051423-9ee43f3a-4fe8-11e7-85c2-f62a659f4085.png)
(Sorry for the bad editing)

**Comment by tamcy** [comment]
Actually I'm not asking for a version as flat as @hfhchan proposed, but a version which angle is similar to the JP glyph. Something like this:

![shs-50-5](https://user-images.githubusercontent.com/959433/27164907-0ee6c0e2-51c3-11e7-91e9-3343673536f3.png)

*(Top to bottom: not-so-good outline recreation from sneak-peek glyph, my suggestion modified from the sneak-peek glyph, current JP version for comparison)*

While I understand that the JP version of 丶 may not fit well with the new HK/TW design of 辶, I still don't think it need to be that... vertical. My 丶 is taken from the JP version of 辶, and slightly rotated clockwise. 

There are some other changes to the original sneak-peek design. I'm sure the designers are already aware of these, and will certainly come up with a much better solution than what I've made (also 迁 is a simplified Chinese character and is rarely used in HK, so this glyph isn't very important to me). But for the sake of completeness I'm going to summarize these changes anyway:

![shs-50-7](https://user-images.githubusercontent.com/959433/27164910-13a182c0-51c3-11e7-83d3-d74249dbc191.png)

1. Although the TW/HK version of 辶 follows the script style and is thus more complicated than the other versions, its nature (as a radical) remains, so the distribution ratio of the components shouldn't be too far away from the other versions. So the left side of 辶 has been scaled down, giving a more harmonic feel to the enclosing component 千 in term of visual heaviness. 
2. When stacking with 千 The left side of 辶 looks too thick or heavy. The height of the horizontal stroke is nearly the same as that for the JP glyph, but as said TW/HK version of 辶 is more complicated. So I've made the strokes thinner to compensate its complexity.
3. A widened 辶 and a compressed-and-shifted 千 causes 辶 to appear larger than 千, making the whole glyph inclined to the right side. Another issue is that it looks like two components are of different gravity horizontally. For 千, the length of the first stroke (丿) too close to the second (一), and such design gives an unnatural feeling to readers. I've kind of "restored" the 千 from JP version and made minor adjustments.
